src/configuration/vimrc.ts

```diff
@@ -68,6 +69,11 @@ export class VimrcImpl {
           VimrcImpl.clearRemapsFromConfig(config, clearRemap);
           continue;
         }
+        const setOptionAction = vimrcSetOptionBuilder.buildSetAction(line);
```

Comment:
In the long run we'd want this to be more general (as many ex-commands can be run from `.vimrc`), but this makes sense in the shorter term

Reply:
Totally agree about that.